src/es/components/organisms/Footer.js

```diff
@@ -252,9 +284,9 @@ export default class Footer extends Shadow() {
    * @return {void}
    */
   renderHTML () {
-    this.loadChildComponents().then(children => Array.from(this.root.querySelectorAll('a')).forEach(a => {
+    this.loadChildComponents().then(children => Array.from(this.root.querySelectorAll('a')).reverse().forEach(a => {
```

Comment:
is there any particular reason to add  reverse() here?

Reply:
don't remember why i added it, but doesnt seem to change anything - removed it 👍